### PR TITLE
Add devfiles for PHP

### PR DESCRIPTION
**Pull request by svor**
### What does this PR do?
Adds devfiles for PHP:
  - [x] [PHP Web Simple](https://github.com/che-samples/web-php-simple)
  - [x] [PHP Laravel](https://github.com/gothinkster/laravel-realworld-example-app)
  - [x] [PHP Symfony](https://github.com/symfony/)
  - [x] [PHP Database Simple Application](https://github.com/taniarascia/pdo/tree/master)

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13528
https://github.com/eclipse/che/issues/13529

### CODE REVIEW

File: devfiles/php-laravel/devfile.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+-
+  name: laravel-realworld-example-app
+  source:
+    type: git
+    location: "https://github.com/gothinkster/laravel-realworld-example-app.git"
+components:
+-
+  type: chePlugin
+  id: redhat/php/latest
+  memoryLimit: 1000Mi
```
Comment (sleshchenko) [inline devfiles/php-laravel/devfile.yaml]: it's not major comment at all, but it could be `1Gi`

File: devfiles/php-laravel/devfile.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+-
+  type: chePlugin
+  id: redhat/php-debugger/latest
+  memoryLimit: 256Mi
+-
+  type: dockerimage
+  alias: tools
```
Comment (sleshchenko) [inline devfiles/php-laravel/devfile.yaml]: is there something except php inside? Can it be named as `php`?

File: devfiles/php-laravel/devfile.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    - name: 'db'
+      port: 3306
+      attributes:
+        discoverable: "true"
+  mountSources: false
+commands:
+-
+  name: Start Artisan Web Server
+  actions:
+  - type: custom
```
Comment (sleshchenko) [inline devfiles/php-laravel/devfile.yaml]: Have you tested it? I wonder why it worked, maybe some validation is missing on workspace creation, but type here must be `exec`

File: devfiles/php-laravel/devfile.yaml (hunk 2 above)
Comment (svor) [reply devfiles/php-laravel/devfile.yaml]: there is nothing special, changed to `php`

File: devfiles/php-laravel/devfile.yaml (hunk 3 above)
Comment (svor) [reply devfiles/php-laravel/devfile.yaml]: I've tested it and it is working. But to have `exec` type for the commands is better solution, changed it

File: devfiles/php-laravel/devfile.yaml (hunk 1 above)
Comment (svor) [reply devfiles/php-laravel/devfile.yaml]: changed to `1Gi`

**Pull request merged**